pkg/v1/mutate/mutate_test.go

```diff
@@ -453,6 +453,11 @@ func TestCanonical(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	for _, h := range cf.History {
+		if h.CreatedBy != "bazel build ..." {
+			t.Errorf("%q != %q", h.CreatedBy, "bazel build ...")
```

Comment:
Can you phrase this error message as `got %q, want %q`? This will help debug the issue if this ever fails.

Reply:
I'm making it identical to the checks below. I think we should keep it similar instead of updating it for this single test assertion.